### ISSUE TITLE
Procedure entry point could not be located when running rtosim_ik_from_nexus.exe and other executables

### ISSUE THREAD

**Issue by jerela** [RealTimeBiomechanics/rtosim#14]
Hi!

After building the solution and attempting to run examples (of my own, because for some reason Vicon Virtual System 1.4.1 couldn't send the provided test data to Vicon Nexus) I get an error when I try to execute rtosim_ik_from_nexus.exe:
![image](https://user-images.githubusercontent.com/61044352/74514998-590b2980-4f16-11ea-811c-c94434b54b74.png)
The error is in Finnish but translates to "The procedure entry point ?replace@ObjectGroup@OpenSim@@QEAAXPEBVObject@2@PEAV32@@Z could not be located in the dynamic link library C:\Program Files (x86)\RTOSIM\bin\rtosim_ik_id_from_nexus.exe".

I get similar errors while trying to execute rtosim_id_from_file.exe, rtosim_ik_from_file.exe and rtosim_ik_id_from_file.exe, although the procedure entry points are slightly different and the dynamic link library refers to the executable I'm attempting to execute.

If you've run into similar issues, please let me know if and how you've solved them.

I didn't follow all of the versions given in the readme because they were outdated for many dependencies. Some of the versions I used were Simbody 3.7 without patching (because patching replaced some files which resulted in Visual Studio being unable to build the solution), Visual Studio 2019 and Vicon Datastream SDK 1.9.0.

Thanks in advance!

**Comment by cpizzolato** [comment]
Hi @jerela. In past I have encountered this problem when the executable was referring to out of date dynamic libraries, which could easily happen if you have manually moved around libraries or executables. But I really have no better answer than that,

For this project I am still working with OpenSim 3.2 and Simbody 3.5.3. In past I had problem using Simbody 3.7 with older versions of OpenSim, but I had some compilation issues, so I don't think it's related. Nonetheless, the idea is doing some maintenance of the current repository and move it to OpenSim4.0+.

**Comment by cpizzolato** [comment]
Hi @jerela I have added a simple example to stream marker data from Vicon Nexus. Could you please check if it works for you?

**Comment by jerela** [comment]
Hi @cpizzolato I'll try reinstalling the package using only recommended software versions whenever possible and see if I can get the example to work. I'll reply here as soon as I've tried that.

**Comment by jerela** [comment]
Hi again @cpizzolato I just built and installed everything again from scratch with the files you updated a few hours ago. I ran the example "exampleViconNexus.exe" and it seemed to work. It created a file "markers.trc" in rtosim-master/tests/testViconNexus/output, which you'll find attached in this reply (as a .txt. file instead of .trc, because I couldn't attach .trc files into this reply).
[markers.txt](https://github.com/RealTimeBiomechanics/rtosim/files/4223956/markers.txt)

The problem I created this issue to address was solved, so I shall close the issue.
